test/common.py

```diff
@@ -2357,6 +2357,8 @@ def run_browser(self, html_file, expected=None, message=None, timeout=None, extr
             if extra_tries > 0:
               print('[test error (see below), automatically retrying]')
               print(e)
+              if '?' not in html_file:
```

Comment:
should we maybe make this '?capture_stdio` not in html_file and/or assert that no other query params are in the URL?

Reply:
My thinking was to leave open the possibility of existing URL parameters in tests.  So this would only work for tests that don't have URL params already.

Reply:
Right, I guess my thought was, if there were other URL params added, then this would silently stop working at some random point in the future (i.e. when there coincidentally happened be an overlap). Adding an assert would make it loudly stop working.

Reply:
OK, I made it more robust.  `run_browser` now assert that no params are pasted and appends the `?capture_stdio` locally each time it runs.